packages/com.sncommunity.advanced-checklist/src/common/utils.ts

```diff
@@ -73,7 +73,7 @@ function createTaskFromLine(rawTask: string): TaskModel | undefined {
     id: uuidv4(),
     description,
     completed: IS_COMPLETED.test(rawTask),
-    createdAt: new Date(),
+    createdAt: new Date().toISOString(),
```

Comment:
How will this work with existing notes?

Reply:
Great question! The "non-serializable values" message from Redux is more of a warning than an error I think, as this was working until now and I only discovered the message by running the plugin locally. Existing notes will still have dates as `createdAt` and will trigger the warning, but the comparison in line 130 in this file should still work okay since doing `new Date()` on a date will do nothing. We could also "migrate" the createdAt values of existing notes to strings, but I'd say it's not worth it -- if I'm understanding correctly, the only thing that lastActive comparison does for now is trigger the autofocus on the input for the last group.